octavia_f5/common/constants.py

```diff
@@ -57,14 +57,10 @@
 # special listener tags
 LISTENER_TAG_NO_SNAT = 'ccloud_special_l4_deactivate_snat'
 
-SINGLE_USE_DH = 'singleUseDh'
-STAPLER_OCSP = 'staplerOCSP'
-TLS_1_0 = 'tls1_0'
-TLS_1_1 = 'tls1_1'
-TLS_1_2 = 'tls1_2'
-TLS_1_3 = 'tls1_3'
-TLS_OPTIONS_SERVER = (SINGLE_USE_DH, STAPLER_OCSP, TLS_1_0, TLS_1_1, TLS_1_2, TLS_1_3)
-TLS_OPTIONS_CLIENT = (SINGLE_USE_DH, TLS_1_0, TLS_1_1, TLS_1_2, TLS_1_3)
+TLS_1_0 = 'TLSv1'
+TLS_1_1 = 'TLSv1.1'
+TLS_1_2 = 'TLSv1.2'
+TLS_1_3 = 'TLSv1.3'
```

Comment:
Why not using the constants from octavia_lib, SSL_VERSION_3, TLS_VERSION_1, etc.

Reply:
Because these constants here were already existing, albeit unused. Why introduce unused constants in the first place :P